Resources/Prototypes/Entities/Objects/Fun/toys.yml

```diff
@@ -605,7 +605,7 @@
   - type: Icon
     sprite: Objects/Fun/toys.rsi
     state: singularitytoy
-  - type: ToySingularity
+  - type: SingularityDistortion
```

Comment:
surely you have to change the fields for the toy distortion, otherwise it'd be the same effect as the regular singularity?

Reply:
Done in https://github.com/space-wizards/space-station-14/pull/4194/commits/17db10fdaf0e540e0d58658f0565a51f5c89b3c4
The values for the toy singularity are different now because the previous ones absolutely ended my game.